frontend/src/scenes/project/Settings/DangerZone.tsx

```diff
@@ -28,19 +28,22 @@ export function DeleteProject(): JSX.Element {
         })
     }
 
-    const DeleteButton = (
-        <Button type="primary" danger onClick={handleClick}>
-            Delete Project
-        </Button>
-    )
-
-    let deletionDisabledReason: string | null = null
+    let accessRestrictionReason: string | null = null
     if ((currentOrganization?.membership_level ?? -1) < OrganizationMembershipLevel.Admin)
-        deletionDisabledReason = 'You must be an organization administrator to delete projects.'
+        accessRestrictionReason = 'This section is restricted to administrators.'
```

Comment:
Well, not strictly admins, since organization owner can do that too, but maybe that's included.

Reply:
Yup I think that in this context by "administrators" you also understand owners.